2/contrib/s2i/assemble

```diff
@@ -17,6 +19,8 @@ if [ -f ${JENKINS_INSTALL_DIR}/plugins.txt ]; then
     echo "Failed to install plugins."
     exit 1
   fi
+  echo "Deleting $INCREMENTAL_BUILD_ARTIFACTS_DIR/plugins"
+  rm -rf "$INCREMENTAL_BUILD_ARTIFACTS_DIR/plugins"
```

Comment:
I think you don't want this inside the `if plugins.txt` block, it should just always be run.  Someone could in theory do an incremental s2i build with a previous image that has installed plugins, but without supplying a `plugins.txt` to the new build iteration, at which point you still want to remove the injected /tmp/artifact content from the newly produced image, even though you didn't invoke `install-plugins.sh`

Reply:
Fixed, thanks!